### DISCUSSION TITLE
Is there a v 1.0.0 somewhere?

### DISCUSSION THREAD

**Discussion by rperpe** [Louisvdw/dbus-serialbattery#597] [Q&A] [answered]
Here and there docs are speaking about a 1.0.0 version resp. >= 1.0.0 and e.g. a config.ini file. Latest release [here](https://github.com/Louisvdw/dbus-serialbattery/releases) seems to be v 0.14.3 from December 2022.

Is there somewhere a 1.0.0+ release available?

**Comment by mr-manuel** [comment, marked as answer]
Not released yet, but available as pre version/nightly: [Install over SSH: nightly version/beta test](https://louisvdw.github.io/dbus-serialbattery/general/install#install-over-ssh-nightly-versionbeta-test)
